ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -538,7 +534,7 @@ public sealed class Bot : IAsyncDisposable, IDisposable {
 			}
 
 			if ((botName.Length > 2) && botName.Contains("..", StringComparison.Ordinal)) {
```

Comment:
This would need a `SharedInfo.ArrayDots.Any(dots => botName.Contains(dots, StringComparison.Ordinal))` instead of what we currently have.

Reply:
Of course if you follow my other suggestion you don't use `ArrayDots` but `RangeIndicators` here ;)

Reply:
This is different, we explicitly want to check against `..`, while other calls require from us an array, and the only reason why it's extracted into a standalone field is to not allocate it each time. It's correct as it is, adding array here is unwanted.

Reply:
Then we should still have it as a `const string` somewhere and make the array reference that.

Reply:
Ah okay, I see the context now, I guess your suggestion makes sense.